azure-pipelines.yml

```diff
@@ -65,7 +65,14 @@ steps:
     Contents: '$(Build.BuildId).zip'
     TargetFolder: '$(Build.ArtifactStagingDirectory)\ArtifactsToBePublished'
 
-- task: PublishBuildArtifacts@1
-  displayName: 'Publish Artifact: drop'
+- task: AzureWebApp@1
```

Comment:
Does this overwrite existing deployment somehow?

Reply:
From what I understand, MSDeploy removes non-matching files at the destination [non-optionally](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-rm-web-app-deployment?view=azure-devops#web-deploy).

> # Remove additional files at destination: 
> Deletes files in the Azure App Service that have no matching files in the App Service artifact package or folder being deployed.